index.html

```diff
@@ -0,0 +1,157 @@
+            <li class="nav-item"><a href="#speakers" class="nav-link">Speakers</a></li>
+            <li class="nav-item"><a href="#program" class="nav-link">Program</a></li>
+            <li class="nav-item"><a href="about.html" class="nav-link">About Us</a></li>
+            <li class="nav-item"><a href="#contact" class="nav-link">Contact</a></li>
+          </ul>
+        </div>
+      </nav>
+      <div class="main-text">
+        <h1 class="main-text__title">Cosmopolitan Affirming Community <br> Global Summit <br> 2023</h1>
+        <p class="main-text__bio">
+          Family is a vital aspect of life, but especially important for the LGBTQI+ community. LGBTQI+ individuals often face rejection and discrimination, making it difficult to find a sense of belonging and acceptance. At <strong>C.A.C</strong> we strive to be a supportive and loving family, biological or chosen, providing emotional and psychological. <br> Welcome To The Family.
+        </p>
+        <h2 class="main-text__dates">2023.06.1<sup>st</sup>(THU) ~ 16<sup>th</sup>(FRI)</h2>
+        <h3 class="main-text__location">
+          <i class="material-icons">place</i>
+          K.I.C.C | Nairobi - Kenya.
+        </h3>
+      </div>
+    </header>
+  </section>
+
+  <!-- Programs section -->
+  <section id="program">
+    <div class="program-intro">
+      <h2 class="program-title">Main Program</h2>
+      <div class="text-separator"></div>
+    </div>
+      <div class="program-container">
+        <div class="program__card">
+          <img src="./elements/lecture.png" alt="Lecture image" class="card-image">
+          <span class="program-card__title">
+            Sensetize
+          </span>
+          <p class="program-card__text">
+            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Molestias doloribus beatae ducimus deserunt dolor distinctio esse eaque, aliquam quasi
+          </p>
+        </div>
+        <div class="program__card">
+          <img src="./elements/Ignite.png" alt="Lecture image" class="card-image">
+          <span class="program-card__title">
+            Unite
+          </span>
+          <p class="program-card__text">
+            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Molestias doloribus beatae ducimus deserunt dolor distinctio esse eaque, aliquam quasi
+          </p>
+        </div>
+        <div class="program__card">
+          <img src="./elements/forum.png" alt="Lecture image" class="card-image">
+          <span class="program-card__title">
+            Research & develop
+          </span>
+          <p class="program-card__text">
+            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Molestias doloribus beatae ducimus deserunt dolor distinctio esse eaque, aliquam quasi
+          </p>
+        </div>
+      </div>
+      <i class="program-link"> <u>SEE THE WHOLE PROGRAM</u></i>
+      <button class="program__button">
+        Be Part C.A.C Summit 2023
+      </button>
+  </section>
```

Comment:
- [ ] Good job creating all the required cards :+1:, but it is requested to have five cards inside your program section :100: 
- [ ] Kindly, you should check the link of the middle image because it seems to be broken
- [ ] While you did a wonderful job creating all your articles :+1: , for semantics you should use an `article` tag for each element inside the "main program" section

**Your section**

![Screenshot from 2023-02-09 06-38-31](https://user-images.githubusercontent.com/105117832/217803781-9f7e28f0-1204-42fe-a52b-c250c635c9e5.png)

**Required section**

![Screenshot from 2023-02-09 06-38-13](https://user-images.githubusercontent.com/105117832/217803859-b84297b4-9868-4eff-9609-179075fb3859.png)